### PR TITLE
Rebalance/port: refactor ported, ointment/brute pack healing nerf but max stack doubled. gauze heals a bit now

### PR DESCRIPTION
**Pull request by ghost**
## Why It's Good For The Game
I think they're a little overpowered but i didn't want to straight nerf them
the tg fix had a negative amount of votes on the PR so i didn't want to bring it here

## Changelog
:cl:

balance: ointments and bruise now heal 15 less, max stack doubled(12, they still spawn in 6)
add: Gauze now heal 5 brute and have half the self application time

/:cl:

### CODE REVIEW

**Comment by Owai-Seek** [comment]
Are you able to still use gauze after someone has stopped bleeding now, or is it only a single use of gauze, no bleeding, 5 (total) brute removal?

First aid packs are fairly rare across the station, and they require targeting specific limbs. (Honestly, in most cases, if you target head or chest it's fine, but for things like shock or spacefaring damage it's a big pain in the ass.)

File: code/game/objects/items/stacks/medical.dm [hunk 1]
```diff
@@ -118,12 +119,13 @@
 
 /obj/item/stack/medical/gauze
 	name = "medical gauze"
-	desc = "A roll of elastic cloth that is extremely effective at stopping bleeding, but does not heal wounds."
+	desc = "A roll of elastic cloth that is extremely effective at stopping bleeding, heals miner's minor wounds."
```
Comment (Ghommie) [inline code/game/objects/items/stacks/medical.dm]:
```suggestion
	desc = "A roll of elastic cloth that is extremely effective at stopping bleeding, heals minor wounds."
```

**Comment by ghost** [comment]
it's the same but it heals 5 brute now when used and it's a little quicker to use it on yourself

**Comment by Arturlang** [comment]
It would be great if you also ported the TG refactors for this, or ill do it and cause conflicts for you. Its needed so brute kids dont heal bloodsuckers.

**Comment by ghost** [comment]
which refactor? is it in their nerf pr?

**Comment by Arturlang** [comment]
Not in the nerf PR, https://github.com/tgstation/tgstation/pull/44415

File: code/game/objects/items/stacks/medical.dm [hunk 2]
```diff
@@ -108,27 +52,56 @@
 	icon_state = "brutepack"
 	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
-	heal_brute = 40
+	var/heal_brute = 25
 	self_delay = 20
-	grind_results = list("styptic_powder" = 10)
+	max_amount = 12
+	grind_results = list(/datum/reagent/medicine/styptic_powder = 10)
```
Comment (Thalpy) [inline code/game/objects/items/stacks/medical.dm]: This won't work here, we haven't ported the reagents datum conversion.

**Comment by Arturlang** [comment]
Awesome, thanks!

File: code/game/objects/items/stacks/medical.dm [hunk 3]
```diff
@@ -163,13 +136,19 @@
...
+	if(iscarbon(M))
+		return heal_carbon(M, user, 0, heal_burn)
+	to_chat(user, "<span class='notice'>You can't heal [M] with the \the [src]!</span>")
+
 /obj/item/stack/medical/ointment/suicide_act(mob/living/user)
 	user.visible_message("<span class='suicide'>[user] is squeezing \the [src] into [user.p_their()] mouth! [user.p_do(TRUE)]n't [user.p_they()] know that stuff is toxic?</span>")
 	return TOXLOSS
-
-/obj/item/stack/medical/get_belt_overlay()
-	return mutable_appearance('icons/obj/clothing/belt_overlays.dmi', "pouch")
```
Comment (Ghommie) [inline code/game/objects/items/stacks/medical.dm:175]: Code is fine and dandy, and I do not mind the healing power / uses trade. Just bring back these couple lines before I merge it, ok?

**Review by Ghommie — APPROVED** [review]

**Pull request merged**